Effects/Filters/source/PlayState.hx

```diff
@@ -131,6 +131,8 @@ class PlayState extends FlxState
 		checkbox.callback = function()
 		{
 			FlxG.camera.filtersEnabled = !checkbox.checked;
+
+			FlxG.game.setFilters(filters);
```

Comment:
I say revert this change and replace line 113 with a setFilters call

Reply:
done